imod/protocols/protocol_tomoNormalization.py

```diff
@@ -286,7 +286,9 @@ def generateOutputStackStep(self, tsObjId):
             newTomogram.copyAttributes(tomo, '_origin')
 
         output.append(newTomogram)
+        output.updateDim()
```

Comment:
This is not longer necessary... You probably rescued it because there was an error in the dimension assignment and this partially fixed it.